pkgs/games/airshipper/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+, zlib
+, sqlite
+, openssl
+, vulkan-loader
+, wayland
+, wayland-protocols
+, libxkbcommon
+, libX11
+, libXrandr
+, libXi
+, libXcursor
+, pkg-config
+, makeWrapper
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "airshipper";
+  version = "0.6.0";
+
+  src = fetchFromGitLab {
+    owner = "Veloren";
+    repo = "airshipper";
+    rev = "v${version}";
+    sha256 = "sha256-m3H2FE1DoV/uk9PGgf9PCagwmWWSQO/gCi7zpS02/WY=";
+  };
+
+  cargoSha256 = "sha256-ddy4TjT/ia+sLBnpwcXBVUzAS07ar+Jjc04KS5/arlU=";
+
+  buildInputs = [
+    openssl
+    libxkbcommon
+    zlib
+    sqlite
+    openssl
+    vulkan-loader
+    wayland
+    wayland-protocols
+    libxkbcommon
+    libX11
+    libXrandr
+    libXi
+    libXcursor
+  ];
+  nativeBuildInputs = [ pkg-config makeWrapper copyDesktopItems ];
+
+  postFixup = ''
+    wrapProgram "$out/bin/airshipper" \
+      --set LD_LIBRARY_PATH "${lib.makeLibraryPath ([ vulkan-loader wayland wayland-protocols libxkbcommon libX11 libXrandr libXi libXcursor ])}"
```

Comment:
I only see this now. We should use patchelf instead.

Reply:
Did it in 3935d8a9d2497424d72d4c774c4851d5c5e19fdc